src/mbgl/renderer/painter.cpp

```diff
@@ -89,7 +89,7 @@ Painter::Painter(gl::Context& context_, const TransformState& state_, float pixe
     tileBorderSegments.emplace_back(0, 0, 4, 5);
     rasterSegments.emplace_back(0, 0, 4, 6);
 
-    gl::debugging::enable();
+    context.enableDebugging();
```

Comment:
Refactored this so it always gets enabled as part of the `Backend`'s `Context` object construction.